### ISSUE TITLE
Featrure request: Download of project from GitHub via link or download from own repositories

### ISSUE THREAD

**Issue by 2Friendly4You** [gitbutlerapp/gitbutler#3457]
I like the feature of GitHub Desktop that I can clone repos from my own GitHub account or with a link and would love if this feature was also available in your application.

<img width="385" alt="Screenshot 2024-04-08 204117" src="https://github.com/gitbutlerapp/gitbutler/assets/83555863/d5758e41-c3de-464b-b417-6e1bbe5d8950">

**Comment by SellersEvan** [comment]
I agree! This feature would be extremely helpful! At least for now, having the ability to clone from a URL would be helpful.

**Comment by Qix-** [comment]
The thing is that GitButler sort of relies on the ability to push to the remote repository, which you inherently lose when you clone as a tarball or a zip (as the Git history isn't typically included). We don't manage branches in the conventional sense so I'm struggling to see what sort of functionality we'd even provide in this case.

This might be something we support in the future but for now I'm going to close this as it's not really how GitButler is intended to work. Sorry about that!
